modules/aws-workspace-with-firewall/providers.tf

```diff
@@ -4,8 +4,10 @@ provider "aws" {
 
 // initialize provider in "MWS" mode to provision new workspace
 provider "databricks" {
-  alias    = "mws"
-  host     = "https://accounts.cloud.databricks.com"
-  username = var.databricks_account_username
-  password = var.databricks_account_password
-}
\ No newline at end of file
+  alias         = "mws"
+  host          = "https://accounts.cloud.databricks.com"
+  username      = var.databricks_account_username
+  password      = var.databricks_account_password
```

Comment:
you either need to use `username`/`password` or `client_id`/`client_secret`, not both